redirect/routes.php

```diff
@@ -29,7 +29,8 @@ class Route {
         ],
         'anal'             => [
             'description' => 'Analysis für Informatiker',
-            'moodle_id'   => '61369',
+            'target'      => 'https://www.moodle.tum.de/course/view.php?id=73098',
+            'moodle_id'   => '73696',
```

Comment:
2 different moodle redirects for Analysis, if that's fine for you: 
anal.tum.sexy -> "Analysis für Informatik" moodle page
manal.tum.sexy -> "Übungen zu Analysis für Informatik" moodle page